src/main/java/com/github/bfh/study/slb/rest/FileUploadService.java

```diff
@@ -0,0 +1,164 @@
+package com.github.bfh.study.slb.rest;
+
+import org.jboss.resteasy.plugins.providers.multipart.InputPart;
+import org.jboss.resteasy.plugins.providers.multipart.MultipartFormDataInput;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.MultivaluedMap;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+import java.io.*;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * File Upload Service
+ *
+ * @author Dario Carosella
+ */
+@Path("file")
+public class FileUploadService {
+
+    private static final Logger _log = LoggerFactory.getLogger(FileUploadService.class);
```

Comment:
do not use underlines. use uppercase

Reply:
uppercase for class variable are for Constants... but logger isn't a constant. So I corrected it to lowercase as you have in the class PrepareStep.java (line: 24)